ui/src/components/Shell/index.jsx

```diff
@@ -92,10 +92,12 @@ export default class Shell extends Component {
           });
 
           this.client.resize(
+            terminal.screenSize.height,
```

Comment:
And it looks like you need to remove the trailing comma after `terminal.screenSize.width` now.